frontend/src/metabase/common/components/QueryColumnPicker/BucketPickerPopover/BaseBucketPickerPopover.styled.tsx

```diff
@@ -45,7 +44,9 @@ export const ChevronDown = styled(Icon)`
   flex: 0 0 auto;
   width: 8px;
   margin-left: 0.25em;
+  color: inherit;
   opacity: 0.75;
+  color: currentColor !important;
```

Comment:
1. `color` is specified twice
2. Previously the icon got highlighted on hover with text, now it does not.